dtschema/schemas/system-idle-states.yaml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright 2025 BayLibre
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/system-idle-states.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: System Idle States
+
+maintainers:
+  - Markus Schneider-Pargmann <msp@baylibre.com>
+
+description: >
+  A system idle state node represents a system state that can have certain
+  limitations and power properties.
+
+properties:
+  $nodename:
+    const: system-idle-states
+
+patternProperties:
+  "^system-":
+    type: object
+    additionalProperties: false
+    description:
+      The node represents a system state description.
+
+    properties:
+      compatible:
+        const: "system-idle-state"
+
+      idle-state-name:
+        $ref: /schemas/types.yaml#/definitions/string
+        description:
+          Name of the system state.
+
```

Comment:
If you do not have any properties here, are you expecting the name to define the type of the state? The idle-state-name in idle-states.yaml is just descriptive, like label. You want the value to be the ABI, right?

Reply:
Yes, this should be ABI. Should I change the description to make this clear?

Reply:
If it is an ABI, 'idle-state-name' should be required as well.